nuxt.config.js

```diff
@@ -134,7 +134,7 @@ export default {
           onlyOnRoot: true, // recommended for SEO
           useCookie: true,
         },
-        langDir: "i18n/locales",
+        langDir: "node_modules/@ourjapanlife/findadoc-localization/locales",
```

Comment:
🤔 I probably should be importing this...

Reply:
Actually I have no idea how this will work 🤨

Reply:
interesting......I think this should work. depends on how it handles paths

Reply:
It does work, It just feels dirty somehow